pykafka/producer.py

```diff
@@ -245,8 +245,11 @@ def produce(self, message, partition_key=None):
             message to
         :type partition_key: bytes
         """
+        if not (isinstance(partition_key, bytes) or partition_key is None):
+            raise TypeError("Producer.produce accepts a bytes object as partition_key, "
+                            "but it got '%s'", type(partition_key))
         if not (isinstance(message, bytes) or message is None):
-            raise TypeError("Producer.produce accepts a bytes object, but it "
+            raise TypeError("Producer.produce accepts a bytes object as message, but it "
```

Comment:
I'm wondering if it's useful to check the opposite for these: `if isinstance(message, unicode)` - so that people could in principle use anything that behaves as `bytes`?